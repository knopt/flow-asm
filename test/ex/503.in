1
9
-4.28504275109868
-5.619704035203943 
8.870836458966721 
3.135719707008114 
1.9356865630250581 
-1.6531596285495898 
1.9974055872026124 
-7.2769486784176385 
4.7662333740759095 
6.395577591300803 

6.6919209069770105 3.6506540701048564 5.711430556358643 7.850156712422606 1.4350112685564405 -3.1248008528587468 -4.867690071649687 4.9295955246278815 -3.142987747711059 
