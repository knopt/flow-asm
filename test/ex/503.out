-160.1920095512324 
140.7822827845638 
-49.742418412351725 
-27.20571423134965 
-80.3233625563713 
111.15722378194556 
-179.0449739727142 
123.86308436268033 
60.532343308216255 
